src/main/java/com/patreon/euphrates/Redshift.java

```diff
@@ -119,17 +126,39 @@ private void createRedshiftPool() {
 
   private void populateTableSizes() {
     try (Connection connection = connectionPool.getConnection()) {
-      try (PreparedStatement statement = connection.prepareStatement("select \"table\", size from svv_table_info where schema = ?")) {
-        statement.setString(1, config.redshift.schema);
-        ResultSet rs = statement.executeQuery();
+      connection.createStatement().execute("create table if not exists euphrates_table_timings (tablename varchar(255), secondsTook int)");
+      try (ResultSet rs = connection.createStatement().executeQuery("select tablename, secondsTook from euphrates_table_timings")) {
         while (rs.next()) {
-          tableSizes.put(rs.getString(1).toLowerCase(), rs.getInt(2));
+          tableSizes.put(rs.getString(1).toLowerCase(), rs.getLong(2));
         }
-        LOG.info(String.format("tableSizes is %s", tableSizes));
+        LOG.info(String.format("table times is %s", tableSizes));
       }
+      connection.commit();
     } catch (SQLException e) {
       throw new RuntimeException(e);
     }
   }
+
+  private void persistTableSizes() {
+    try {
+      try (Connection connection = connectionPool.getConnection()) {
+        PreparedStatement updateStatement = connection.prepareStatement("update euphrates_table_timings set secondsTook = ? where tablename = ?");
+        PreparedStatement insertStatement = connection.prepareStatement("insert into euphrates_table_timings (secondsTook, tablename) values (?, ?)");
+
+        LOG.info(String.format("new table times is %s", newTableSizes));
```

Comment:
Can't these log calls just be `LOG.info("new table times is {}", newTableSizes)`?

Reply:
Oh yeah, they all could be. I'll fix this later (or maybe you will?) :)